sdk/digitaltwins/azure-digitaltwins-core/src/main/java/module-info.java

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module come.azure.digitaltwins.core {
+    requires transitive com.azure.core;
+    requires com.fasterxml.jackson.annotation;
+    requires com.azure.http.netty;
+
+    opens com.azure.digitaltwins.core to come.azure.core, com.azure.http.netty;
+    opens com.azure.digitaltwins.core.models to com.fasterxml.jackson.annotation, come.azure.core;
+    opens com.azure.digitaltwins.core.util to com.fasterxml.jackson.annotation, come.azure.core;
+    opens com.azure.digitaltwins.core.serialization to com.fasterxml.jackson.annotation, come.azure.core;
+    opens com.azure.digitaltwins.core.implementation to com.fasterxml.jackson.annotation, come.azure.core;
+    opens com.azure.digitaltwins.core.implementation.models to com.fasterxml.jackson.annotation, com.azure.core;
+    opens com.azure.digitaltwins.core.implementation.serializer to com.fasterxml.jackson.annotation, com.azure.core;
+    opens com.azure.digitaltwins.core.implementation.converters to com.fasterxml.jackson.annotation, com.azure.core;
```

Comment:
how did you come up with this list? is there a script that lists them?

Reply:
Unfortunately there is none ! I used the cheat sheet and my own judgement along with help from @alzimmermsft